machines/Issuers/IssuersMachine.ts

```diff
@@ -496,4 +535,5 @@ export interface issuerType {
   credential_configurations_supported: object;
   display: [displayType];
   credentialTypes: [CredentialTypes];
+  authorizationEndpoint: string;
```

Comment:
authorization_servers cam be removed as not in use now

Reply:
Done